src/applications/letters/actions/letters.js

```diff
@@ -61,20 +61,24 @@ export function getLetterList(dispatch) {
       if (status === '403') {
         // Backend authentication problem
         dispatch({ type: BACKEND_AUTHENTICATION_ERROR });
+        throw new Error(`vets_letters_error_getLetterList: ${status}`);
```

Comment:
:thinking: I'm pretty sure this will result in the same grouping in Sentry as we have right now.

While perusing the Raven docs, I found that you can [fingerprint](https://docs.sentry.io/data-management/rollups/?platform=javascript#custom-grouping) for custom grouping. What do you think about doing that right before we throw the error?

Reply:
Updated. It would be nice to have a way to verify this, though. Any ideas on how to do that?

Reply:
Could you manually send an event in the dev environment and see how it shows up?